observations/unhealthy-scale-out-mid-deploy.md

```diff
@@ -0,0 +1,67 @@
+# What happens when a scale-out event occurs mid-deployment of an unhealthy artifact?
+
+In this test we deployed [application version](../dist) `ABC` to `500` in the stack
+[`ScalingAsgRollingUpdate`](../packages/cdk/lib/scaling-asg-rolling-update.ts) (CFN stack `playground-CODE-scaling-asg-rolling-update`).
+
+The aim of this test is to understand the behaviour in the ["Ophan scenario"](https://github.com/guardian/riff-raff/issues/1342).
```

Comment:
Minor: I don't think this aim is applicable in this scenario?

```suggestion
```

Reply:
Hmm, not sure I agree. Any deployment can fail, and we can scale at any time. To me, "Ophan scenario" is "we need to scale during deployment", there's no restriction on if the artifact being deployed is good or bad.

TBH I was tempted to merge the two documents together, but it didn't read very clearly.